script/generate-red-worker-code.js

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env node
+
+/**
+ * This script takes the transpiled JavaScript code for `RedundantAudioEncoder` and places all of that code into a
+ * string that can be used to run a worker. 
+ */
+
+const fs = require('fs');
+const RedundantAudioEncoder = require('../build/redundantaudioencoder/RedundantAudioEncoder.js').default;
+
+const debugLine = 'RedundantAudioEncoder.shouldLogDebug = true;';
+const withDebug = `${RedundantAudioEncoder.toString()}
+${debugLine}
+RedundantAudioEncoder.shouldReportStats = true;
+RedundantAudioEncoder.initializeWorker();
+`.replace(/"/g, '\'').replace(/\n/g, "\\n")
```

Comment:
Where does the `"` come from?

Reply:
Some of the comments in `RedundantAudioEncoder` class use double quotes. I can change the class to only use single quotes, which will remove the need for that replacement.

Reply:
Yeah I think for styling we always use ' in this repo so probably want to update the original file also.

Reply:
Sure, I'll leave the regex there for safety, but I'll change all of the double quotes in the original file to single quotes.